### ISSUE TITLE
How to generate a architecute model with torch

### ISSUE THREAD

**Issue by NLGithubWP** [D-X-Y/NATS-Bench#42]
In the README.MD, there is only one example to generate an architecture at index 12 with get_cell_based_tiny_net.

But in the codebase, I found there are many functions including:
1. get_cell_based_tiny_net
2. obtain_model

Which is the real model at index 12? (The one whose performance is recorded in get_more_info).

If I want to obtain the model/architecture at the index 12 whose performance is the exactly measured and recorded in get_more_info. Which method should i use ? 

Thank you

**Comment by D-X-Y** [comment]
Thanks for your questions, `print(api.arch(12))` can tell you the architecture info, see more details in the updated README (https://github.com/D-X-Y/NATS-Bench#usage).

For the second question, you can see item-3 in (https://github.com/D-X-Y/NATS-Bench#usage), while a prerequisite is to download the pre-trained checkpoints, which is very large..

**Comment by NLGithubWP** [comment]
> Thanks for your questions, `print(api.arch(12))` can tell you the architecture info, see more details in the updated README (https://github.com/D-X-Y/NATS-Bench#usage).
> 
> For the second question, you can see item-3 in (https://github.com/D-X-Y/NATS-Bench#usage), while a prerequisite is to download the pre-trained checkpoints, which is very large..

**Comment by NLGithubWP** [comment]
Thank you for th eansw

**Comment by NLGithubWP** [comment]
Thank you for the answering, 

`print(api.arch(12))` could give us the cell structure. 

The architecture/model is composed by stacking cell multiple times. 

I want to get the accuracy and loss of the 12 model for example (get_more_info already give us that ), and also get the full model itself. And then I want to retrain the model with my own datasets.  I want to check if the model with high accuracy has also good performance on my datasets. 

But i'm still a little bit confuse with generating network related APIs. 

```
import xautodl
from xautodl.models import get_cell_based_tiny_net
config = api.get_net_config(15620, 'cifar10')
info = api.get_more_info(15620, 'cifar10')
network = get_cell_based_tiny_net(config)
```

In this code, it uses 'get_cell_based_tiny_net' to get the network.  And use `get_more_info` to get the loss and accuracy of the model. 

Is the network returned by `get_cell_based_tiny_net` can give me the full model ? It is quite small and accuracy is low. 

In xautod/models package, I also see there are `get_cifar_models` apis, which can return us  basic, resnet, and densenet model.

How to use them ? And which one should

Thank you

**Comment by sumegha1024** [comment]
Hello. I tried replicating the above. However info['test-accuracy'] gives a different result from running the trained network on the test data. To extract test data I used.

**Comment by sumegha1024** [comment]
@KelvinYang0320 I tried replicating the above. However info['test-accuracy'] gives a different result from running the trained network on the test data. To extract test data I used 
```
valid_data = dset.CIFAR10(root='path to input data', train=False, download=True, transform=valid_transform). 
valid_queue = torch.utils.data.DataLoader(
valid_data, batch_size=len(valid_data), shuffle=False, pin_memory=True)
```
I then used the network you defined above along with valid_queue to calculate test_accuracy. However my calculated accuracy of 83.8100 is closer to the validation accuracy while info['test-accuracy'] returns 86.25. I also made sure I used the same transform functions.

Did you check and manage to replicate the test accuracy? If yes, could you confirm if the test data and data loader that I use is correct or perhaps share a screenshot of how you replicated the result? Thanks a lot!

**Comment by D-X-Y** [comment]
Thanks, @KelvinYang0320, for help answer the question!
